scripts/image_upload_glance.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python2
+
+"""
+Script to upload iso/ova/qcow images to Glance server.
+This script requires that python-glanceclient be installed.
+
+pip install python-glanceclient
+
+Usage:
+1.scripts/image_upload_glance.py --image cfme-rhevm-5.8.2.1-1.x86_64.qcow2 \
+                                 --image_name_in_glance cfme-5821.qcow2 \
+                                 --provider glance11
+
+"""
+import argparse
+import sys
+
+from cfme.utils.conf import cfme_data, credentials
+from keystoneauth1 import loading
+from keystoneauth1 import session
+from glanceclient import Client
+
+
+def upload_to_glance(image, image_name_in_glance, provider):
+    """
+    Upload iso/qcow images to Glance.
+    """
+    api_version = '2'  # python-glanceclient API version
+    provider_dict = cfme_data['management_systems'][provider]
+
+    loader = loading.get_plugin_loader('password')
+    auth = loader.load_from_options(
+        auth_url=provider_dict['auth_url'],
+        username=credentials['cloudqe_rhos_default']['username'],
```

Comment:
**Required**
This key should come from `provider_dict.credentials`, not be hard coded.